flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/debezium/task/context/StatefulTaskContext.java

```diff
@@ -184,6 +185,51 @@ private MySqlOffsetContext loadStartingOffsetState(
     }
 
     private boolean isBinlogAvailable(MySqlOffsetContext offset) {
+        String gtidStr = offset.gtidSet();
+        if (gtidStr != null) {
+            if (gtidStr.trim().isEmpty()) {
+                return true; // start at beginning ...
+            }
+            String availableGtidStr = connection.knownGtidSet();
+            if (availableGtidStr == null || availableGtidStr.trim().isEmpty()) {
+                // Last offsets had GTIDs but the server does not use them ...
+                LOG.info(
+                        "Connector used GTIDs previously, but MySQL does not know of any GTIDs or they are not enabled");
+                return false;
+            }
+            // GTIDs are enabled
+            GtidSet gtidSet = new GtidSet(gtidStr);
+            // Get the GTID set that is available in the server ...
+            GtidSet availableGtidSet = new GtidSet(availableGtidStr);
+            if (gtidSet.isContainedWithin(availableGtidSet)) {
+                LOG.info(
+                        "MySQL current GTID set {} does contain the GTID set required by the connector {}",
+                        availableGtidSet,
+                        gtidSet);
+                final GtidSet knownServerSet = availableGtidSet;
+                final GtidSet gtidSetToReplicate =
```

Comment:
IIUC, the variable ```gtidSetToReplicate``` meaning the remaing gtid to process?
It's nothing do with replication.
So I think the name for the variable ```gtidSetToReplicate``` is a little of confused.
Maybe  ```gtidSetToProcess``` is better?

Reply:
thanks, i think `Replicate` is easier to understand.  The replication is concept of mysql master-slave replication protocol. 
In this logic, it means find subset of master-slave shoud replicate.

Reply:
Ok, make sense to me.

Reply:
thanks, i will do it.